packages/pagination/src/PaginationButtons.stories.tsx

```diff
@@ -1,17 +1,6 @@
 import { ComponentStory, ComponentMeta } from '@storybook/react';
 import { useState } from 'react';
-import useSWR from 'swr';
```

Comment:
This table with remote data example has been moved out of the pagination stories and into the `examples` section